### ISSUE TITLE
xrdcl.unzip ZIP Central Directory corrupted, but zip ok

### ISSUE THREAD

**Issue by rodwalker** [xrootd/xrootd#2207]
Hi,
just for this file, and not xroot version dependent this time,
xrdcp root://xrootd-at2-door.pic.es:1094//pnfs/pic.es/IFAEAtlasTier2/t2atlasdatadisk/rucio/data17_13TeV/0a/88/DRAW_RPVLL.20624245._000730.zip.1?xrdcl.unzip=DRAW_RPVLL.20624243._009629.pool.root.1 .
[0B/0B][100%][==================================================][0B/s]  
Run: [ERROR] Received corrupted data: ZIP Central Directory corrupted. (source)
but the downloaded zip can be unpacked fine, e.g.
$ zip -T DRAW_RPVLL.20624245._000730.zip.1
test of DRAW_RPVLL.20624245._000730.zip.1 OK

I can put the zip file somewhere, if you want to look.

Cheers,
Rod.

**Comment by amadio** [comment]
@rodwalker Yes, please, if you could put it somewhere on EOS at CERN, then I can access. Thank you!

**Comment by rodwalker** [comment]
Hi,
It is already there at
| data17_13TeV | DRAW_RPVLL.20624245._000730.zip.1 | 2.961 GB   | 5a900fcb  | CERN-PROD_DATADISK: root://eosatlas.cern.ch:1094//eos/atlas/atlasdatadisk/rucio/data17_13TeV/0a/88/DRAW_RPVLL.20624245._000730.zip.1                                                                                                                            |

Cheers,
Rod.
